src/registrar/assets/js/get-gov-admin.js

```diff
@@ -608,11 +598,13 @@ document.addEventListener('DOMContentLoaded', function() {
 
     modalConfirm.addEventListener("click", () => {
         textarea.removeAttribute('readonly');
+        textarea.focus();
```

Comment:
Thanks! @gabydisarli this shifts the focus to the textbox when you successfully click edit. I thought this was needed since the edit btn disappears and the UI does not show any visible feedback.

Reply:
I like that a lot. Thank you.